be/src/exprs/rpc_fn_call.cpp

```diff
@@ -0,0 +1,332 @@
+            ptype->set_id(PGenericType::DATE);
+            const auto* time_val = (const DateTimeValue*)(src_slot);
+            PDateTime* date_time = arg->add_datetime_value();
+            date_time->set_day(time_val->day());
+            date_time->set_month(time_val->month());
+            date_time->set_year(time_val->year());
+            break;
+        }
+        case TYPE_DATETIME: {
+            ptype->set_id(PGenericType::DATETIME);
+            const auto* time_val = (const DateTimeValue*)(src_slot);
+            PDateTime* date_time = arg->add_datetime_value();
+            date_time->set_day(time_val->day());
+            date_time->set_month(time_val->month());
+            date_time->set_year(time_val->year());
+            date_time->set_hour(time_val->hour());
+            date_time->set_minute(time_val->minute());
+            date_time->set_second(time_val->second());
+            date_time->set_microsecond(time_val->microsecond());
+            break;
+        }
+        case TYPE_TIME: {
+            ptype->set_id(PGenericType::DATETIME);
+            const auto* time_val = (const DateTimeValue*)(src_slot);
+            PDateTime* date_time = arg->add_datetime_value();
+            date_time->set_hour(time_val->hour());
+            date_time->set_minute(time_val->minute());
+            date_time->set_second(time_val->second());
+            date_time->set_microsecond(time_val->microsecond());
+            break;
+        }
+        default: {
+            FunctionContext* fn_ctx = context->fn_context(_fn_context_index);
+            fn_ctx->set_error(
+                    fmt::format("data time not supported: {}", _children[i]->type().type).c_str());
+            break;
+        }
+        }
+    }
+
+    brpc::Controller cntl;
+    _client->fn_call(&cntl, &request, response, nullptr);
+    if (cntl.Failed()) {
+        FunctionContext* fn_ctx = context->fn_context(_fn_context_index);
+        fn_ctx->set_error(cntl.ErrorText().c_str());
+        return Status::InternalError(fmt::format("call to rpc function {} failed: {}",
+                                                 _rpc_function_symbol, cntl.ErrorText())
+                                             .c_str());
+    }
+    if (response->status().status_code() != 0) {
+        FunctionContext* fn_ctx = context->fn_context(_fn_context_index);
+        fn_ctx->set_error(response->status().DebugString().c_str());
+        return Status::InternalError(fmt::format("call to rpc function {} failed: {}",
+                                                 _rpc_function_symbol,
+                                                 response->status().DebugString()));
+    }
+    return Status::OK();
+}
+
+template <typename T>
+T RPCFnCall::interpret_eval(ExprContext* context, TupleRow* row) {
+    T res_val;
+    PFunctionCallResponse response;
+    Status st = _eval_children(context, row, &response);
+    if (!st.ok() || response.status().status_code() != 0 ||
```

Comment:
The situation of `response.status().status_code() != 0` has been checked in `_eval_children()`.
No need to check again here.